README.md

```diff
@@ -1,5 +1,9 @@
 # healthchecks
 
+[![Travis build badge](https://img.shields.io/travis/andela/healthchecks-thunknalla.svg)]()
```

Comment:
Update this badge to use the markdown format of the badge from Travis in place of this one from shields.